packages/checkbox/theme/lumo/vaadin-checkbox-styles.js

```diff
@@ -30,6 +31,8 @@ registerStyles(
     }
 
     [part='checkbox'] {
+      width: var(--vaadin-checkbox-size, var(--_checkbox-size));
+      height: var(--vaadin-checkbox-size, var(--_checkbox-size));
```

Comment:
Eventually, it would be great to be able to avoid reassigning these properties, and only rely on the `--vaadin-checkbox-size` property. Would there be a way to give a “deprecation warning” if developers use the `--lumo-size-m` property to change the size of the checkbox?

Reply:
> Would there be a way to give a “deprecation warning” if developers use the --lumo-size-m property to change the size of the checkbox?

I'm afraid not. The current code looks weird but I couldn't come up with a cleaner approach 🤷‍♂️ 
Do you think we could proceed with it, and cherry-pick also to `23.2` and `23.3` branches?

Reply:
Thanks. If you don't have any other suggestions, I will merge the PR.